blog/2024-01-28-reductstore-v1_8_0-released/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "ReductStore v1.8.0 Has Been Released with Data Replication"
+description: ReductStore v1.8.0 introduces data replication between ReductStore instances.
+authors: alexey
+tags: [news]
+slug: news/reductstore-8-released
+date: 2024-01-28
+---
+
+# ReductStore v1.8.0 Has Been Released with Data Replication
+
+We are pleased to announce the release of the latest minor version of [ReductStore](https://www.reduct.store/), [1.8.0](https://github.com/reductstore/reductstore/releases/tag/v1.7.0). ReductStore is a time series database designed for storing and managing large amounts of blob data.
+
+To download the latest released version, please visit our [**Download Page**](https://www.reduct.store/download).
+
+## What's New in ReductStore v1.8.0
+
+In this release, we've introduced a crucial feature for any database - data replication. Now, you can create a server-side task that "subscribes" to new records written to a bucket and forwards them to another bucket. This bucket can be located on the same instance or a remote one. Since all databases implement replication differently based on their specializations, let's examine how ReductStore tackles this.
+
+<!--more-->
+{/* truncate */}
+
+### Replication in ReductStore
+
+#### Append-Only Approach
+
+Currently, replication is append-only, meaning we only synchronize new records when they are written. As ReductStore is typically used as a large ring-buffer for blob data, it automatically removes old data when the disk quota is exceeded. Therefore, it doesn't make sense to replicate the remove operation on a remote instance where the disk space might be different.
```

Comment:
I would clarify the "we only synchronize new records when they are written" and already say here that new records are copied to the destination buckets but not deleted or updated.

Reply:
The "large ring-buffer" could also be clarified by mentionning that it's an efficient FIFO buffer